src/components/CallView/shared/EmptyCallView.vue

```diff
@@ -114,7 +122,10 @@ export default {
 			if (this.isConnecting) {
 				return t('spreed', 'Connecting …')
 			}
-			return t('spreed', 'Waiting for others to join the call …')
+			if (this.isOneToOneConversation()) {
+				return t('spreed', 'Waiting for {user} to join the call', { user: this.getDisplayName() })
```

Comment:
Error:   126:71  error  Use this.getDisplayName instead of this.getDisplayName()                  vue/no-use-computed-property-like-method